### PR TITLE
Clean up eclipse and github actions and switch gradle to require Java 11

### PR DESCRIPTION
**Pull request by jhanders34**
- Update to remove non existent source files from jpa client fat
- Add test directory to the .classpath  for accesslists.internal project
- Update github actions test categories for spelling error and new test suite
- Change to require gradle builds to use Java 11
- Update Java 11 project to reference Java 11 settings in .classpath, bnd.bnd and org.eclipse.jdt.core.prefs files
- Update io.openliberty.concurrent projects to be Java 11 since they are only used for concurrency 3.0 features.
- Change to compile against jta 1.3.3 instead of 1.1 for the jdbc 4.3 fat so that it builds in eclipse

### CODE REVIEW

**Review by njr-11 — APPROVED** [review]

**Review by tevans78 — APPROVED** [review]

File: dev/settings.gradle [hunk 1]
```diff
@@ -165,12 +165,14 @@ pluginManagement {
...
 if (!JavaVersion.current().isJava11Compatible()) {
-  println "WARNING: Building this repository will soon require Java 11. To use Java 11 follow these steps:\n" + 
+  println "ERROR: Building this repository requires Java 11. To use Java 11 follow these steps:\n" + 
                             "  1) You can download a copy for your OS here if you don't already have it: " + 
-                            "https://adoptopenjdk.net/index.html?variant=openjdk11&jvmVariant=openj9\n" + 
+                            "http://ibm.biz/GetSemeru\n" + 
                             "  2) run 'export JAVA_HOME=/path/to/your/java11' in the shell, or set in ~/.bashrc\n" + 
                             "  3) [MACOS ONLY] Set 'ulimit -Sn 1024' in your ~/.bashrc\n" + 
                             "  4) Restart your gradle daemon with './gradlew --stop'\n";
+        throw new GradleException('Curent Java version is not 11 or higher.')
```
Comment (cbridgha) [inline dev/settings.gradle]: sp check on "current"

**Review by cbridgha — APPROVED** [review]
Looks good - next we should check if there is a gradle / bnd upgrade that wasn't possible with us running Java 8

**Comment by jhanders34** [comment]
#build

File: dev/settings.gradle (hunk 1 above)
Comment (jhanders34) [reply dev/settings.gradle]: fixed

**Comment by jhanders34** [comment]
#libby

**Pull request merged**